README.md

```diff
@@ -56,6 +56,22 @@ cargo run --release --features "risc0 succinct"
 RISC0_DEV_MODE=1 cargo run --release --features risc0
 ```
 
+#### Bonsai
+```
+# edit run_bonsai.sh and run
```

Comment:
Can you change this so it's possible to simply use bonsai by doing `prove_block.sh testnet risc0-bonsai 10`. So just add an additional proof type that sets everything up as expected.

I want to be lazy and make it as easy as possible to run the different provers.

Reply:
Of course